migrations/m190412_115343_xcoin_product_optimization.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m190412_115343_xcoin_product_optimization
+ */
+class m190412_115343_xcoin_product_optimization extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->renameColumn('xcoin_product','payment_type', 'offer_type');
+        $this->alterColumn('xcoin_product', 'price', $this->float(4));
```

Comment:
what is `float(4)` ? 
I always tried to store prices as integer values in the smallest denomination to prevent floating point/rounding issues.

I'd allow more thean 255 chars for a comment, can't we simply use a text there?

Reply:
`float(4)` is for float precision. Do you think that we must change this to integer ? Wouldn't that be a problem for users who want prices with floating point ? 

You're right for the comment field but i already talked with @KarimChabrak and we think to remove it or to just change it by a `select` with ` Per Unit, Per Hour, Per Day, Per Service ` as options.

Reply:
ok, then we should remove it now (without adding something else) if we do not use that? 

re integer: I'd only store it as integer and convert it in the app. e.g. store in milim and then show and use dinar in the app. (so multiply or divide by 1000 in the app) 
for EUR values that would be then cents and multiply or divide by 100 in the app) 

but that's cool now! more fyi. what I always did